paddle/fluid/operators/tensorrt/tensorrt_engine_op.h

```diff
@@ -168,7 +247,8 @@ class TensorRTEngineOp : public framework::OperatorBase {
 
   void Prepare(const framework::Scope &scope, const platform::Place &dev_place,
                TensorRTEngine *engine) const {
-    VLOG(4) << "Prepare engine";
+    LOG(INFO) << "Prepare TRT engine (Optimize model structure, Select OP "
```

Comment:
VLOG

Reply:
During the process of generating the calibration table, i think there is some necessary information that is good for users（user shoud not set `export GLOG_v=3` to display those info ）.
 
Such as：
```
I0116 03:40:57.720394 10328 tensorrt_engine_op.h:132] Running calibration trt int8 ...
I0116 03:40:57.720635 10335 tensorrt_engine_op.h:250] Prepare TRT engine (Optimize model structure, Select OP kernel etc). This process may cost a lot of time.
I0116 03:40:58.782788 10328 tensorrt_engine_op.h:132] Running calibration trt int8 ...
I0116 03:40:58.889819 10328 tensorrt_engine_op.h:132] Running calibration trt int8 ...
I0116 03:40:58.996390 10328 tensorrt_engine_op.h:132] Running calibration trt int8 ...
I0116 03:40:59.103065 10328 tensorrt_engine_op.h:132] Running calibration trt int8 ...
I0116 03:40:59.209903 10328 tensorrt_engine_op.h:132] Running calibration trt int8 ...
I0116 03:40:59.320547 10328 tensorrt_engine_op.h:132] Running calibration trt int8 ...
I0116 03:40:59.428273 10328 tensorrt_engine_op.h:132] Running calibration trt int8 ...
I0116 03:40:59.535076 10328 tensorrt_engine_op.h:132] Running calibration trt int8 ...
I0116 03:40:59.641471 10328 tensorrt_engine_op.h:132] Running calibration trt int8 ...
I0116 03:40:59.748912 10328 tensorrt_engine_op.h:132] Running calibration trt int8 ...
I0116 03:40:59.855603 10328 analysis_predictor.cc:581] Wait for calib threads done.
I0116 03:40:59.855656 10328 analysis_predictor.cc:583] Finish wait.

I0116 03:41:20.101413 10328 analysis_predictor.cc:597] Write Paddle-TRT INT8 calibration table data to file ./mobilenetv1/trt_calib_6034075461045681804

```